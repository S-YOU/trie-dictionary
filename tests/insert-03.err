nodes: 2
nodes: 3
nodes: 4
nodes: 5
nodes: 6
nodes: 7
nodes: 8
nodes: 9
nodes: 10
nodes: 11
nodes: 12
nodes: 13
nodes: 14
nodes: 15
nodes: 16
nodes: 17
nodes: 18
nodes: 19
nodes: 20
nodes: 21
nodes: 22
nodes: 23
nodes: 24
nodes: 25
nodes: 26
nodes: 27
