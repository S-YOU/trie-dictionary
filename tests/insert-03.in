insert a
insert c
insert e
insert g
insert i
insert k
insert m
insert o
insert q
insert s
insert u
insert w
insert y
insert b
insert d
insert f
insert h
insert j
insert l
insert n
insert p
insert r
insert t
insert v
insert x
insert z
