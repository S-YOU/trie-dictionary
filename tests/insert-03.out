word number: 0
word number: 1
word number: 2
word number: 3
word number: 4
word number: 5
word number: 6
word number: 7
word number: 8
word number: 9
word number: 10
word number: 11
word number: 12
word number: 13
word number: 14
word number: 15
word number: 16
word number: 17
word number: 18
word number: 19
word number: 20
word number: 21
word number: 22
word number: 23
word number: 24
word number: 25
